<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MastersWnd</class>
 <widget class="QWidget" name="MastersWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="5" rowspan="3">
    <widget class="QPushButton" name="addPb">
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="2">
    <widget class="QLineEdit" name="nameLE"/>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="adressLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="6">
    <widget class="QTableView" name="mastersTv"/>
   </item>
   <item row="3" column="2" rowspan="2">
    <widget class="QLabel" name="doneWorkLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="numberLE"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="numberLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" rowspan="2">
    <widget class="QLabel" name="priceLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QLabel" name="surenameLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="QLineEdit" name="surenameLE"/>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QLabel" name="nameLb">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="adressLE"/>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="doneWorkLE"/>
   </item>
   <item row="2" column="3">
    <widget class="QLineEdit" name="priceLE"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
